package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "integreat-react-native-app",
-  "version": "2020.3.1",
+  "version": "0.1",
```

Comment:
Change this to `0.1.0`. Else this is not a valid version for NPM or Yarn. If an invalid version is used then it is not possible to use this in a monorepo :P